medusa/notifiers/slack.py

```diff
@@ -92,7 +92,7 @@ def _send_slack(self, message=None, webhook=None):
 
         headers = {b'Content-Type': b'application/json'}
         try:
-            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot')), headers=headers)
+            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot', icon_url='https://raw.githubusercontent.com/pymedusa/Medusa/master/static/images/ico/favicon-310.png')), headers=headers)
```

Comment:
We may want to use a copy from the CDN.

Reply:
Yeah I figured, I wasn't sure what the best image to use would be